apps/test/unit/templates/teacherDashboard/SectionTableTest.js

```diff
@@ -33,90 +37,122 @@ describe('SectionTable', () => {
-    assert.equal(courseLink, '//localhost-studio.code.org:3000/courses/csd');
-    assert.equal(sectionLink, '//localhost-studio.code.org:3000/s/csd1');
-  });
+    it('gradeFormatter has grade text', () => {
+      const rowData = sectionSet[0];
+      const gradeCol = shallow(gradeFormatter(null, {rowData}));
+      const text = gradeCol.find('div').at(0).text();
+      assert.equal('5', text);
+    });
 
-  it('courseLinkFormatter contains course text and section text', () => {
-    const rowData = sectionSet[0];
-    const courseLinkCol = shallow(courseLinkFormatter(null, {rowData}));
-    const courseText = courseLinkCol.find('a').at(0).text();
-    const sectionText = courseLinkCol.find('a').at(1).text();
-    assert.equal(courseText, 'CS Discoveries');
-    assert.equal(sectionText, 'Unit 1: Problem Solving');
-  });
+    it('courseLinkFormatter provides links to course information and section information', () => {
+      const rowData = sectionSet[0];
+      const courseLinkCol = shallow(courseLinkFormatter(null, {rowData}));
+      const courseLink = courseLinkCol.find('a').at(0).props().href;
+      const sectionLink = courseLinkCol.find('a').at(1).props().href;
+      assert.equal(courseLink, '//localhost-studio.code.org:3000/courses/csd');
+      assert.equal(sectionLink, '//localhost-studio.code.org:3000/s/csd1');
+    });
 
-  it('sectionLinkFormatter contains section link', () => {
-    const rowData = sectionSet[0];
-    const sectionLinkCol = shallow(sectionLinkFormatter(null, {rowData}));
-    const sectionLink = sectionLinkCol.prop('href');
-    assert.equal(sectionLink, pegasus('/teacher-dashboard#/sections/1'));
-  });
+    it('courseLinkFormatter contains course text and section text', () => {
+      const rowData = sectionSet[0];
+      const courseLinkCol = shallow(courseLinkFormatter(null, {rowData}));
+      const courseText = courseLinkCol.find('a').at(0).text();
+      const sectionText = courseLinkCol.find('a').at(1).text();
+      assert.equal(courseText, 'CS Discoveries');
+      assert.equal(sectionText, 'Unit 1: Problem Solving');
+    });
+
+    it('sectionLinkFormatter contains section link', () => {
+      const rowData = sectionSet[0];
+      const sectionLinkCol = shallow(sectionLinkFormatter(null, {rowData}));
+      const sectionLink = sectionLinkCol.prop('href');
+      assert.equal(sectionLink, pegasus('/teacher-dashboard#/sections/1'));
+    });
 
-  it('sectionLinkFormatter contains section text', () => {
-    const rowData = sectionSet[0];
-    const sectionLinkCol = shallow(sectionLinkFormatter(null, {rowData}));
-    const sectionText = sectionLinkCol.text();
-    assert.equal(sectionText, 'sectionA');
+    it('sectionLinkFormatter contains section text', () => {
+      const rowData = sectionSet[0];
+      const sectionLinkCol = shallow(sectionLinkFormatter(null, {rowData}));
+      const sectionText = sectionLinkCol.text();
+      assert.equal(sectionText, 'sectionA');
+    });
```

Comment:
A lot of whitespace changes here because I like to nest all tests under a single describe matching the name of the module (i.e. SectionTable in this case).